docs: update docs for useCart composable (#353)
vuestorefront/bigcommerce: docs/composables/use-cart.md

## Features

The `useCart` composable can be used to:

- load cart information,
- add, update and remove items in the cart,
- checking if the product is already added to the cart.

## API

> - `cart: Cart` - a main data object. See [Cart](../reference/api/bigcommerce-api.cart.md) for more information.

Reviewer: ```suggestion
- `cart: Cart` - a main data object. See the [Cart](../reference/api/bigcommerce-api.cart.md) interface for more information.
```

> - `load` - function for fetching cart from a server or creating a brand new if it doesn't exist. This method populates the `cart` object with the result. This method accepts a single object with following properties:

Reviewer: ```suggestion
- `load` - function for fetching cart from the API or creating a new one if it doesn't already exist. This method populates the `cart` object with the result. This method accepts a single object parameter with the following properties:
```

    };
    ```

> - `addItem` - function for adding products to the cart. This method populates the `cart` object with the updated result. This method accepts a single object with the following properites:

Reviewer: ```suggestion
- `addItem` - function for adding products to the cart. This method populates the `cart` object with the updated result and accepts a single object parameter with the following properties:
```

>   - `product: Product` - See [Product](../reference/api/bigcommerce-api.product.md) for more information.

Reviewer: ```suggestion
  - `product: Product` - See the [Product](../reference/api/bigcommerce-api.product.md) interface for more information.
```

    };
    ```

> - `updateItemQty` - function for updating a quantity of a product already in the cart. This method populates the `cart` object with the updated result. This method accepts a single object with following properites:

Reviewer: ```suggestion
- `updateItemQty` - function for updating a quantity of a product already in the cart. This method populates the `cart` object with the updated result and accepts a single object parameter with the following properties:
```

>   - `product: CartItem` - See [CartItem](../reference/api/bigcommerce-api.cartitem.md) for more information.

Reviewer: ```suggestion
  - `product: CartItem` - See the [CartItem](../reference/api/bigcommerce-api.cartitem.md) interface for more information.
```

  - `quantity: number`

> - `removeItem` - function for removing a product that currently is in the cart. This method populates the `cart` object with the updated result. This method accepts a single object with following properites:

Reviewer: ```suggestion
- `removeItem` - function for removing a product that currently is in the cart. This method populates the `cart` object with the updated result and accepts a single object parameter with the following properties:
```

>   - `product: CartItem` - See [CartItem](../reference/api/bigcommerce-api.cartitem.md) for more information.

Reviewer: ```suggestion
  - `product: CartItem` - See the [CartItem](../reference/api/bigcommerce-api.cartitem.md) interface for more information.
```

> - `isInCart` - function for checking if a product is currently in the cart. This method accepts a single object with following properties:

Reviewer: ```suggestion
- `isInCart` - function for checking if a product is currently in the cart. This method accepts a single object with the following properties:
```

>   - `product: Product` - See [Product](../reference/api/bigcommerce-api.product.md) for more information.

Reviewer: ```suggestion
  - `product: Product` - See the [Product](../reference/api/bigcommerce-api.product.md) interface for more information.
```

> - `clear` - a function for removing all items currently stored in the cart. This method populates the `cart` object with the updated result.

Reviewer: ```suggestion
- `clear` - a function for removing all items from the cart. This method populates the `cart` object with the updated result.
```

- `loading: boolean` - a reactive object containing information about the loading state of the cart.

> - `error: UseCartErrors` - reactive object containing the error message if some method failed for any reason.

Reviewer: ```suggestion
- `error: UseCartErrors` - reactive object containing the error message if any of the methods above failed for any reason.
```
